isic-archive-tasks/isic_archive_tasks/image.py

```diff
@@ -0,0 +1,199 @@
+import io
+import os
+import re
+import subprocess
+import tempfile
+import time
+
+from bson import ObjectId
+from celery import chord
+from celery.utils.log import get_task_logger
+import requests
+import sentry_sdk
+
+from girder.models.file import File
+
+from isic_archive_tasks import app, CredentialedGirderTask
+
+
+SUPERPIXEL_VERSION = 3.0
+logger = get_task_logger(__name__)
+
+
+@app.task(base=CredentialedGirderTask)
+def ingestImage(imageId):
+    from girder.plugins.isic_archive.models.image import Image
+    image = Image().load(imageId, force=True)
+
+    if image['ingested']:
+        logger.warning('Attempting to ingest an already ingested image.')
+        return
+
+    try:
+        imageData = Image().imageData(image)
```

Comment:
Can we only wrap these first 3 lines in the `try` block? They are the only ones that are expected to sometimes fail with non-image files.